climada/hazard/trop_cyclone.py

```diff
@@ -402,11 +408,18 @@ def compute_windfields(track, centroids, model):
 
     # restrict to centroids in rectangular bounding box around track
     track_centr_msk = _close_centroids(t_lat, t_lon, centroids)
-    track_centr_idx = track_centr_msk.nonzero()[0]
     track_centr = centroids[track_centr_msk]
 
+    nreachable = track_centr.shape[0]
+    npositions = t_lat.shape[0]
+    windfields = np.zeros((npositions, nreachable, 2))
+    reachable_centroids = track_centr_msk.nonzero()[0]
```

Comment:
Isn't np.array.nonzero() deprecated? 

_Deprecated since version 0.24.0: Please use .to_numpy().nonzero() as a replacement._

Reply:
Maybe also use `reachable_centroids_idx` to indicate that it is the indexes of the centroids and not the centroids themselves. Would have to adjust the variable downstream too.

Reply:
This is a Pandas warning and I have no idea how you end up having a Pandas Series at this point?! Can you give a code example?

Reply:
Oh yeah, sorry got confused ^^. All good.